File: julia8024/BOJ_1789.py
# 백준
# 1789. 수들의 합

S = int(input())

cnt = 0 # N
sum = 0

for i in range(1, S+1):
    sum += i
    cnt = i
    if S < sum:
        cnt -= 1
        break

print(cnt)

File: chaeryeon823/BOJ_1789.js
const fs = require('fs');

const n = BigInt(fs.readFileSync(0, 'utf8').toString().trim());

let sum = 0n;
let i = 1n;
let cnt = 0n;

while (true) {
  if (sum > n) {
    break;
  } else {
    sum += i;
    i += 1n;
    cnt += 1n;
  }
}
console.log(String(cnt - 1n));
